packages/generators/add-generator.ts

```diff
@@ -162,7 +161,7 @@ export default class AddGenerator extends Generator {
 							this.configuration.config.item = action;
 						});
 				}
-				const temp: string = action;
+				const temp = action;
```

Comment:
It doesn't look obvious to me. I would keep the type here

Reply:
`action` is already a string, so `temp` inherits its type